@utahdts/utah-design-system/react/contexts/UtahDesignSystemContext/hooks/useBanner.js

```diff
@@ -0,0 +1,66 @@
+// @ts-check
+import { useCallback, useMemo } from 'react';
+import useUtahDesignSystemContext from '../useUtahDesignSystemContext';
+import uuidv4 from '../../../util/uuidv4';
+
+/** @typedef {import('../../../jsDocTypes').UtahDesignSystemContextBanner} UtahDesignSystemContextBanner */
+/** @typedef {import('../../../jsDocTypes').BannerPlacement} BannerPlacement */
+/**
+ * @returns {{
+ * addBanner: function({
+ *  className: string | undefined,
```

Comment:
use square brackets instead of | undefined so that they are optional

Reply:
e.g. do this, ignore above.

duration?: number,